setup/kubernetes/k3s.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "K3s"
+description: Set up K3s on an Ubuntu machine to deploy Coder.
+---
+
+This article will show you how to install K3s onto a new Ubuntu 20.04 LTS
+machine for use with Coder.
+
+[K3s](https://k3s.io/) is a lightweight Kubernetes distribution that works well
+for single-node or multi-node clusters. This guide covers the installation
+of K3s onto a new Ubuntu 20.04 LTS machine, but if you want to install Coder on
+a local machine or an existing host, [Kind](./kind.md) or [k3d](https://k3d.io/)
```

Comment:
It's unfortunate that there aren't longer descriptions for `kind` or `k3d`, because very short text links like this can be hard to click on mobile devices. Also, it seems that kind is now stylized in all lowercase (at one point it was KinD for kubernetes-in-docker, but maybe they're moving away from a name like that, now that they have podman support)

Reply:
I changed to `kind cluster` and `k3d cluster` so it's easier to navigate on mobile. I also added a "why" these would be a better choice.